source/releasenotes/2024-02-14-q1-product-announcement.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Q1 product announcement - WebOps Certification, NextJS Starter Kits, and more!"
+published_date: "2024-02-14"
+categories: [new-feature, front-end-sites, performance, wordpress, general]
+---
+
+Hope this release note finds you in high spirits. We've got some fantastic news to share from the Q1 product announcement – new features, improvements, and even some goodies for our beloved Pantheon community.
+
+🚀 **WebOps Certification Program and free study guide**
+
+Elevate your skills with our [certification program](https://pantheon.io/certification). Check out the [study guide](https://docs.pantheon.io/certification/study-guide) to get started.
+
+🌐 **NextJS open source starter kits**
+
+Dive into the world of NextJS with our open-source [starter kits](https://decoupledkit.pantheon.io/docs). Kickstart your projects with ease!
+
+🔄 **Front-End Sites Multidev delete and preview feature**
+
+Managing Multidev environments just got smoother. Enjoy enhanced control over your front-end sites.
+
+🚀 **83% faster backups with Pantheon File System**
+
+Say goodbye to slow backups! Our revamped Pantheon File System ensures faster performance.
+
+📊 **File System Status Report for performance issue alerts**
+
+Stay ahead of performance issues with the [File System Status Report](https://docs.pantheon.io/guides/filesystem/large-files#alerts?utm_medium=email&utm_source=newsletter&utm_content=what%27s_new&utm_campaign=2024_02_WEB_ALL_US_Whats_New_Announcement_Q1) available in the site dashboard's "Status" tab.
+
+🌐 **WordPress enterprise-grade database support**
+
+Data-heavy sites, rejoice! Experience enterprise-grade database support for WordPress.
+
+🌐 **Improved multiple domain management experience**
+
+Managing domains just got easier, faster and more intuitive. 
+
+🚀 **PHP 8.3 support**
+
+We now support PHP 8.3. While PHP 8.2 is recommended for now, rest assured that we're ready to support your migration as core CMSes and plugins catch up.
+
+[Learn more about what's new in Q1](https://pantheon.io/about/whats-new).
+
+
+
+
+
+📅 **Pricing updates starting March 4**
+
+Get ready for preferred pricing, 4 months overage protection for qualifying non-profits, and a 40% traffic limit increase for our annual subscribers.
```

Comment:
```suggestion
Get ready for preferred pricing, four months overage protection for qualifying non-profits, and a 40% traffic limit increase for our annual subscribers.
```

Reply:
Yes I included links to the What's New page (https://pantheon.io/about/whats-new) and the policy/pricing changes page (https://pantheon.io/blog/enhanced-pantheon-overage-policy-traffic-limits-and-pricing?utm_medium=email&utm_source=newsletter&utm_content=what%27s_new&utm_campaign=2024_02_WEB_ALL_US_Whats_New_Announcement_Q1).